README.md

```diff
@@ -12,8 +12,51 @@ Please, post an issue on GitHub and **provide a plunker** with your question.
 ```javascript
 npm install ng2-eonasdan-datetimepicker --save
 ```
-
-2) Add the A2Edatetimepicker module import in your module.:
+2) Include all required scripts if it doesn't included yet in your project:
+ - jquery.js, 
+ - bootstrap (.js & .css),
+ - eonasdan bootsrap datetimepicker (.js & .css),
+ 
+ - moment.js.
+
+   (nb: you don't need to do another npm install, the previous command already did it)
+    #### Angular cli:
+    an example by using angular cli, you can add theses libs in scripts and styles keys in  **.angular-cli.json**
+
+
+    ```json 
```

Comment:
From what I understand you are using `angular-cli` yourself. So This part is tested on you, right?

Reply:
that's right. Both are tested.

Reply:
OK I will fix it and merge this changes.